translations/pl.ts

```diff
@@ -247,17 +247,17 @@
     </message>
     <message>
         <source>Elevation</source>
-        <translation type="unfinished"></translation>
+        <translation>Przewyższenie</translation>
     </message>
     <message>
         <source>%1 m a.s.l.</source>
         <extracomment>elevation - meters above sea level</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Przewyższenie - m.n.p.m.</translation>
     </message>
     <message>
         <source>Created</source>
         <extracomment>date time when waypoint was created</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>data utworzenia punktu drogi</translation>
```

Comment:
`extracomment` is just comment for translators, you should translate `source`... :-)

Reply:
sorry, my mistake

Reply:
Would not be just "Utworzenia" better?

![Snimek_20190805_001](https://user-images.githubusercontent.com/309458/62440544-f2c87180-b750-11e9-86ad-02b95d0b0bb8.png)

Reply:
I again translated extracomment. I will review my last commit today.